components/03-organisms/menu/utility-nav/_utility-nav.scss

```diff
@@ -0,0 +1,89 @@
+@use '../../../00-tokens/index.scss' as base;
+@use '../../../01-atoms/lists/list';
+
+.utility-bar {
+  --link-color-base: var(--color-text);
+  --utility-bar-gap: var(--size-spacing-7);
+
+  // Color schemes
+  [data-component-theme='white'] &,
```

Comment:
This feels like it should be bumped down to line 26 as they are styles/definitions for a variation.

Reply:
These were removed entirely with the new menu.link so I'm resolving.